tests/rspec_webui_tests.pm

```diff
@@ -4,8 +4,8 @@ use testapi;
 
 sub run() {
         my $branch = "master";
-        if ($VERSION != "Unstable") {
-          $branch = $VERSION;
+        if (get_var('VERSION') != "Unstable") {
```

Comment:
Thanks for the hint @coolo.

Just wondering why does it then work like expected?
maser: https://openqa.opensuse.org/tests/391616#step/rspec_webui_tests/1
vs. 
2.8: https://openqa.opensuse.org/tests/390453#step/rspec_webui_tests/1